### DISCUSSION TITLE
SSD1331 OLED

### DISCUSSION THREAD

**Discussion by IsaNocNoc** [micropython/micropython#12584] [RP2040 / Pico]
Hi, I have a problem: I want to use the OLED ssd1331 but I can't find any micropython library that I can use; Do you know of a library or any code that works in micropython for the pico?

**Comment by andrewleech** [comment]
When I put "ssd1331 micropython" into Google the first two links were a forum post describing ways to use it and this driver designed for it:
https://github.com/peterhinch/micropython-nano-gui/blob/master/drivers/ssd1331/ssd1331.py
